cvat/apps/engine/task.py

```diff
@@ -345,6 +353,23 @@ def count_files(file_mapping, counter):
 
     return client_video or server_video, client_archive or server_archive
 
+def _download_data(urls, upload_dir):
+    job = rq.get_current_job()
+    local_files = []
+    for url in urls:
+        name = os.path.basename(urlrequest.url2pathname(urlparse.urlparse(url).path))
+        if local_files.count(name) > 0:
```

Comment:
Complexity for count is O(n). Thus complexity for the loop is O(n^2). It is better to use a dictionary here.